Dockerfile

```diff
@@ -20,8 +21,8 @@ RUN apt-get update && \
                     libnss3 \
                     gvfs-bin \
                     xdg-utils \
-                    python -y --no-install-recommends && \
-    apt-get clean && \
+                    python && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* \
```

Comment:
Damn! I should I have at least build it. I forgot you put it in a sole layer. Corrected.